airbyte-ci/connectors/pipelines/pipelines/commands/groups/connectors.py

```diff
@@ -509,8 +516,10 @@ def list(
 
 
 @connectors.command(name="format", cls=DaggerPipelineCommand, help="Autoformat connector code.")
+@click.option("--commit-and-push", default=False)
+@click.option("--export-to-host", default=True)
 @click.pass_context
-def format_code(ctx: click.Context) -> bool:
+def format_code(ctx: click.Context, commit_and_push, export_to_host) -> bool:
```

Comment:
🤔 What if we just remove this instead of update since were relying on `gradlew format` atm

Reply:
Yes, these new options a remainder of an attempt to consolidate these command changing connector code and making them commit and push. I'll revert this. And will open a separate PR to remove format